scripts/buildDocs.sh

```diff
@@ -5,8 +5,28 @@ cd "$(dirname -- "$0")/.."
 
 echo "Changed directory to $(pwd)"
 
+# Build image if not existent
+if [[ "$(docker images -q squidfunk/mkdocs-material:8.5.9 2> /dev/null)" == "" ]]; then
```

Comment:
`squidfunk/mkdocs-material:8.5.9` is used several times, so might be possible to be stored in a variable so declaration only happens once